README.md

```diff
@@ -8,18 +24,51 @@ However, for consistency, we will refer to the output as a BOM.
 
 The project is still very early stages, so the output may as of yet be of little use.
 
-## Using Genealogos
-### Analyzing a local flake
+## Installing Genealogos-cli
+Right now, Genealogos is only distributed via our Nix Flake.
+Ensure you have the experimental features `flakes` and `nix-command` enabled, and then run:
+```fish
+# To run once
+nix run github:tweag/genealogos -- --help
+
+# To temporarily add to $PATH
+nix shell github:tweag/genealogos
+```
+
+You can also forgo Nix and use cargo directly instead.
+However, in this case you are sponsible for ensuring the dependencies are available.
+```fish
+cargo install --git https://github.com/tweag/genealogos.git genealogos-cli
+```
+
+## Getting Started
+### Prerequisites
+Development of Genealogos requires Cargo, and some other dependencies.
+We additionally recommend the use of `rust-analyzer`, `cargo-hack`, and `nixtract`.
+The easiest way to get these tools it through the Nix development environment.
+```fish
+nix develop
+````
+
+Alternatively you can install the dependencies manually, but in that case you are on your own.
+
+### Building
+Due to [a bug in Cargo](https://github.com/rust-lang/cargo/issues/4463), running `cargo [build, check, clippy, etc]` will fail.
+There are two ways to solve this, either by specifying a specific package `cargo COMMAND -p [genealogos, genealogos-cli, genealogos-api]` or by running `cargo hack COMMAND`.
+
+## Usage
+### `genealogos-cli`
+Analyzing a local flake:
 ```fish
 genealogos --flake-ref /path/to/your/local/flake
 ```
 
-### Analyzing `hello` from nixpkgs
+Analyzing `hello` from nixpkgs:
 ```fish
 genealogos --flake-ref nixpkgs --attribute-path hello
 ```
 
-### Using a trace file
+Using a trace file:
 This section assumes you are using the latest `main` version version of [nixtract][nixtract].
```

Comment:
Yes! It is still relevant! This way, you could cache parts of the evaluation. It's a nice feature.

You just need to use the latest version, since we both maintain nixtract and genealogos, we always ensure they are compatible.